torch/testing/_internal/common_nn.py

```diff
@@ -2117,7 +2117,7 @@ def fractional_max_pool3d_test(test_case):
         cudnn=True,
         check_with_long_tensor=True,
         with_tf32=True,
-        tf32_precision=0.005,
+        tf32_precision=0.05,
```

Comment:
do you know if those tests are even run with any datatype other than double (for gradcheck)?

Reply:
I think it runs on many data types, at `test_nn.py`.